lib/excess_flow/sliding_window_strategy.rb

```diff
@@ -60,12 +60,16 @@ def current_requests
       @current_requests ||= ExcessFlow.redis { |r| r.zcount(configuration.counter_key, '-inf', '+inf') }
     end
 
+    def current_ttl
+      @current_ttl ||= ExcessFlow.redis { |r| r.ttl(configuration.counter_key) }.to_i
+    end
+
     def current_timestamp
       @current_timestamp ||= (Time.now.to_f * 100_000).to_i
     end
 
     def start_expiration_window
-      return if current_requests.zero?
+      return if current_ttl.zero?
```

Comment:
Your description of the `ttl` returns values made sense to me... so I don't understand why this checks for `current_ttl.zero?` instead of `current_ttl.negative?` Maybe this strategy is different somehow?

Reply:
Oh yeah nice catch! It was just me being stupid, fixed that